main/remote-states/editor-options.ts

```diff
@@ -9,7 +9,8 @@ import {prettifyFormat} from '../utils/formats';
 const exportUsageHistory = new Store<{[key in Format]: {lastUsed: number; plugins: Record<string, number>}}>({
   name: 'export-usage-history',
   defaults: {
-    gif: {lastUsed: 5, plugins: {default: 1}},
+    gif: {lastUsed: 6, plugins: {default: 1}},
+    hevc: {lastUsed: 5, plugins: {default: 1}},
```

Comment:
I think we should add this after webm, to avoid similar issues we had with AV1

Thoughts on the order? @sindresorhus

Reply:
Agreed, at least as long as Chrome doesn't support HEVC playback.

Reply:
Fixed in 086b988.